### PR TITLE
Pin BinaryNinja version to 3.2.3814.

### PR DESCRIPTION
**Pull request by whyitfor**
**One sentence summary of this PR (This should go in the CHANGELOG!)**
Pin BinaryNinja version to 3.2.3814 in Docker build.

**Please describe the changes in your request.**
Pin BinaryNinja version to 3.2.3814. This is currently the recommended BinaryNinja version for compatibility with OFRAK.

### CODE REVIEW

**Review by EdwardLarson — CHANGES_REQUESTED** [review]
Is the full license.dat required to run the `pin_version.py` script? I don't see a clear place there where it is.

Either way, it looks like the section in the `environment-setup.md` doc needs to be updated with the info about running this file. I think the ofrak_binary_ninja user guide doesn't necessarily need to be updated, but it could be helpful to note that there is a specific recommended supported version.

**Comment by whyitfor** [comment]
> Is the full license.dat required to run the `pin_version.py` script? I don't see a clear place there where it is.
> 

Yes it is -- you cannot use the `binaryninja` python package without a license file.

> Either way, it looks like the section in the `environment-setup.md` doc needs to be updated with the info about running this file. I think the ofrak_binary_ninja user guide doesn't necessarily need to be updated, but it could be helpful to note that there is a specific recommended supported version.

I've updated the environment setup and BinaryNinja guide (the command to build the Docker image is still the same, but there are other updates that were needed).

File: docs/environment-setup.md [hunk 1]
```diff
@@ -128,11 +128,11 @@ docker exec \
 - To manually stop it, run `python -m ofrak_ghidra.server stop`. 
 - Ghidra logs can be found here: `/root/.ghidra/.ghidra_10.1.2_PUBLIC/application.log`.
 
-See [the Ghidra user guide](https://ofrak.com/docs/user-guide/ghidra.html) for more information about using Ghidra with OFRAK.
+See [the Ghidra user guide](./user-guide/ghidra.md) for more information about using Ghidra with OFRAK.
 
 ### Binary Ninja
 
-Note that Binary Ninja is not distributed with OFRAK. Instead, if a license is present, the Docker build step will run the official headless installer using the provided license. **You need to have a valid BinaryNinja license to build and run the image.** For more details, [read the script that is run](https://github.com/redballoonsecurity/ofrak/blob/master/disassemblers/ofrak_binary_ninja/install_binary_ninja_headless_linux.sh).
+Note that Binary Ninja is not distributed with OFRAK. **You need to have a valid headless BinaryNinja license to build and run the image.** For more details, see the [Docker commands that are run](https://github.com/redballoonsecurity/ofrak/blob/master/disassemblers/ofrak_binary_ninja/Dockerstub).
```
Comment (EdwardLarson) [inline docs/environment-setup.md:135]:
```suggestion
Note that Binary Ninja is not distributed with OFRAK. **You need to have a valid headless BinaryNinja license to build and run the image.** The `license.dat` file should be placed in a `.binaryninja` directory, under the user's home directory. For more details, see the [Docker commands that are run](https://github.com/redballoonsecurity/ofrak/blob/master/disassemblers/ofrak_binary_ninja/Dockerstub).
```

**Review by andresito00 — APPROVED** [review]
Take Ed's change and LGTM. Thanks.

**Review by EdwardLarson — APPROVED** [review]

**Pull request merged**